resources/deploy/config.yaml

```diff
@@ -17,58 +13,17 @@ storage:
   matrixstore: ./matrices
   archive_dir: ./examples/archives
   workspaces:
-    default: # required, no filters applied, this folder is not watched
+    default: 
       path: ./examples/internal_studies/
-    # other workspaces can be added
-    # if a directory is to be ignored by the watcher, place a file named AW_NO_SCAN inside
-    tmp:
+    studies:
       path: ./examples/studies/
-      # filter_in: ['.*'] # default to '.*'
-      # filter_out: [] # default to empty
-      # groups: [] # default empty
 
 launcher:
-  default: local
   local:
     binaries:
-      700: path/to/700
-#  slurm:
-#    local_workspace: path/to/workspace
-#    username: username
-#    hostname: 0.0.0.0
-#    port: 22
-#    private_key_file: path/to/key
-#    key_password: key_password
-#    password: password_is_optional_but_necessary_if_key_is_absent
-#    default_wait_time: 900
```

Comment:
The `scripts/package_antares_web.sh` script uses the regex pattern '700: path\/to\/700' to locate this line and replace it with the latest version and solver path. If you modify this value, the search using this regex will no longer work.

I believe the best solution would be to simultaneously modify the `config.yaml` file to include substituable fields and implement a different regex in the `package_antares_web.sh` script.

I suggest the following YAML configuration:

```yaml
launcher:
  local:
    binaries:
      VER: ANTARES_SOLVER_PATH
```

Additionally, you can modify the regex as follows:

```sh
if [[ "$OSTYPE" == "msys"* ]]; then
  sed -i "s/VER: ANTARES_SOLVER_PATH/$ANTARES_SOLVER_FULL_VERSION_INT: .\/AntaresWeb\/antares_solver\/antares-$ANTARES_SOLVER_VERSION-solver.exe/g" "${DIST_DIR}/config.yaml"
else
  sed -i "s/VER: ANTARES_SOLVER_PATH/$ANTARES_SOLVER_FULL_VERSION_INT: .\/AntaresWeb\/antares_solver\/antares-$ANTARES_SOLVER_VERSION-solver/g" "${DIST_DIR}/config.yaml"
fi
```